.env

```diff
@@ -1,6 +1,7 @@
 # DO NOT UPDATE THIS FILE MANUALLY
 # To update this file please run "./scripts/configureApplication"
 
+AFFILIATE_BOOKINGCOM=null
```

Comment:
just a sidenote: I'm thinking about some doc

Reply:
Updated - please check it. Please note that the whole Vault process is still in progress. I already have the Vault namespace, but it's empty and `scripts/configureApplication.js` is not ready as well.